src/bluebottle.h

```diff
@@ -627,7 +628,7 @@ extern real lamb_cut;
  *  greater than 1e-2.
  ******
  */
-
+extern real osci_f;
```

Comment:
Can you add a comment about what this is?

Reply:
This is the frequency to generate a oscillating pressure gradient in z direction. Actually this line doesn't come from me. It should be Gedi's work. "osci_f" is only used in domain.c: 886 and cuda_bluebottle.cu: 1848. Maybe these lines should be deleted if we don't want to keep this feature.